content/post/424-Grml-new-stable-release-2025.08-available.md

```diff
@@ -0,0 +1,44 @@
+---
+author: Chris Hofstaedtler
+categories: []
+date: Sat, 16 Aug 2025 10:00:00 +0200
```

Comment:
not sure whether we manage to have all our mirrors up2date by then, other than that, looks perfect for me! :) :bow: :heart: